natgeo/mortar
# [SYTO-1169] Added Secondary Nav to Color Page

To test:

- Go to Color page
- Verify it contains a left rail menu
- Verify all links are working correctly.
